nodeup/pkg/model/kubelet.go

```diff
@@ -146,6 +148,16 @@ func (b *KubeletBuilder) buildSystemdEnvironmentFile(kubeletConfig *kops.Kubelet
 	if b.UsesCNI() {
 		flags += " --cni-bin-dir=" + b.CNIBinDir()
 		flags += " --cni-conf-dir=" + b.CNIConfDir()
+		// If we are using the AmazonVPC plugin we need to bind the kubelet to the local ipv4 address
+		if b.Cluster.Spec.Networking.AmazonVPC != nil {
```

Comment:
Could we have this be a config flag that gets turned on by default if it's the Amazon CNI? The reason for this is that similar CNIs such as Lyft's CNI will also need this. I would like to give the user the option of setting this through here even if the CNI is not one we currently provide with Kops. This will also make is so that we can reuse this if we ever on-board other CNIs with the same requirement.

Reply:
@rdrgmnzs this is a cherry pick of a PR that is already done.  I am not certain if @justinsb wants separate cherry picks ... open question.  You mind filing an issue?

Reply:
Would you consider it a requirement for this PR to be included in 1.9 or could it be added afterwards? I would be happy to work on it but I am not sure how long it would take me and I'd love for this change to be released with 1.9 since we rely on it.